spring-aot/src/main/resources/META-INF/spring.factories

```diff
@@ -31,6 +31,7 @@ org.springframework.web.WebNativeConfigurationProcessor,\
 org.springframework.core.SynthesizedAnnotationNativeConfigurationProcessor,\
 org.springframework.core.IndexedBeanHierarchyNativeConfigurationProcessor,\
 org.springframework.context.annotation.ScopeNativeConfigurationProcessor
+org.springframework.cloud.function.FunctionTypeProcessor
```

Comment:
@olegz, @sdeleuze  I think there was a mistake after merging this PR, there is a missing comma after org.springframework.context.annotation.ScopeNativeConfigurationProcessor